tamr_toolbox/notifications/emails.py

```diff
@@ -0,0 +1,205 @@
+"""Tasks related to creation of Email notifications"""
+import logging
+import time
+import smtplib
+import ssl
+
+from typing import Union, List, Optional
+from email.mime.text import MIMEText
+from tamr_toolbox.notifications.email_info import EmailInfo
+from smtplib import SMTPException
+
+from tamr_unify_client import Client
+from tamr_unify_client.operation import Operation
+
+from tamr_toolbox.models.operation_state import OperationState
+from tamr_toolbox.utils.operation import get_details, from_resource_id
+
+LOGGER = logging.getLogger(__name__)  # To Do: does additional logging to be added?
+
+
+def build_message(
+    *, message: str, subject_line: str, sender: str, recipients: List[str]
+) -> MIMEText:
+    """Builds email message
+
+    Args:
+        message: Body of email message
+        subject_line: subject of email
+        sender: email address of sender
+        recipients: list of emails to send message to
+
+    Returns:
+        Email as a string
+    """
+
+    # build email
+    msg = MIMEText(message)
+    msg["Subject"] = subject_line
+    msg["From"] = sender
+    msg["To"] = ", ".join(recipients)
+    return msg.as_string()
+
+
+def send_email(
+    *, message: str, subject_line: str, email_config: EmailInfo, raise_error: bool = True,
```

Comment:
I thought it would be annoying to pass the smtp configurations and addresses every time you called send email or monitor job. I can remove the dataclass if having the 5 individual parameters is more intuitive or better

Reply:
I made this change